website/docs/cloud-docs/api-docs/team-tokens.mdx

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
Rather than creating a brand new page for the new API, I decided to break the existing team token API documentation into two sections: "Team tokens API reference" and "Legacy team tokens API reference." Since both APIs will exist simultaneously, I didn't want to make the API tokens site navigation confusing, break existing references to this documentation, or create confusion for users who regularly use these docs. Furthermore, both the old and new API share a lot of the same background information and keeping the two coupled and reusing some of that information felt like the more straightforward thing to do. This all being said, though, I don't feel strongly about coupling or decoupling the two APIs and would love to hear what other folks think!